conduit/data/datasets/audio/ecoacoustics.py

```diff
@@ -41,6 +42,12 @@ class SoundscapeAttr(Enum):
     time = auto()
     NN = auto()
     N0 = auto()
+    A_Den = auto()
+
+
+@enum_name_str
+class MDAttr(Enum):
+    fileName = auto()
```

Comment:
This isn't really a review, just a comment, that this mix of enum naming styles makes me feel quite anxious.

Reply:
as in `MD` as opposed to `Md`?

Reply:
what I mean is that between these two enums, they have members with names
- `habitat`
- `site`
- `time`
- `NN`
- `N0`
- `A_Den`
- `fileName`

In an ideal world there would be some consistency, even though it's not really required for 🐍

Reply:
that should have been resolved at source, i.e. in the csv files, but you deal with what you have.

Reply:
I think when the values correspond to column names it's probably best to just manually write the mappings rathr than relying on the decorator (which simply makes str(.) return .name.lower())

Reply:
so I'll change `fileName` to `file_name`, that will work. I could also change `NN` and `N0` to `species_abundance` and `species_richness`?

Reply:
not sure if this will have a cascade effect, but I assume not, proposal for naming:

```
@enum_name_str
class SoundscapeAttr(Enum):
    HABITAT = auto()
    SITE = auto()
    TIME = auto()
    SPECIES_ABUNDANCE = "NN"
    SPECIES_RICHNESS = "N0"
    A_DEN = "A.Den"
  
  
@enum_name_str
class MDAttr(Enum):
    FILE_NAME = "fileName"
```